### PR TITLE
XRT-1188 branch

### PR DESCRIPTION
**Pull request by Tylerg2471**
* Updated to add S7 example configs and commands

### CODE REVIEW

File: DeviceServices/s7/deployment/config/logger.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "Name":"console",
+  "Level":"Debug"
```
Comment (dpazj) [inline DeviceServices/s7/deployment/config/logger.json:3]: This should be "Level": "Warn"

**Review by dpazj — CHANGES_REQUESTED** [review]
One small comment. Can you also update the title of the PR to describe what is going in?

File: DeviceServices/s7/deployment/config/logger.json (hunk 1 above)
Comment (Tylerg2471) [reply DeviceServices/s7/deployment/config/logger.json]: Fixed now

**Review by dpazj — APPROVED** [review]
LGTM

**Pull request merged**